### DISCUSSION TITLE
append class to row

### DISCUSSION THREAD

**Discussion by swarakaka** [filamentphp/filament#1481] [Help]
Can classes be added rows at a table?

This example is manually inserted through the browser:
![Bildschirmfoto 2022-02-07 um 11 31 44](https://user-images.githubusercontent.com/9349190/152771267-4fb4e144-b782-496a-8585-2d1503ebb2ab.png)

**Comment by studiowizjo** [comment]
I join this request. It would be nice to add method (which accepts also callbacks) for additional rows styling. There is quite often need for visual distinction of certain rows, not only cells.
